code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -786,7 +786,7 @@
 	bayonet_state = "bayonet"
 	knife_x_offset = 22
 	knife_y_offset = 21
-	fire_sound = 'sound/f13weapons/varmint_rifle.ogg'
+	fire_sound = 'sound/f13weapons/assaultrifle_fire.ogg'
```

Comment:
Give  'sound/f13weapons/combatrifle.ogg' a listen, I don't think the assaultrifle_fire.ogg which is just used for automatic weapons would be good for the service rifle.

Reply:
That sounds powerful enough as well.